src/main/java/org/asamk/signal/commands/SetContactNameCommand.java

```diff
@@ -0,0 +1,34 @@
+package org.asamk.signal.commands;
+
+import net.sourceforge.argparse4j.impl.Arguments;
+import net.sourceforge.argparse4j.inf.Namespace;
+import net.sourceforge.argparse4j.inf.Subparser;
+import org.asamk.signal.manager.Manager;
+
+public class SetContactNameCommand implements LocalCommand {
```

Comment:
I'd prefer this command to be more general, like `UpdateContactCommand`. That way it could be extended in the future to update other contact infos, like e.g. the contact's color.

I'd suggest to make the `name` argument a named parameter (`-n`, `--name`) but `.required(true)` for now.
So usage would be like this: `signal-cli ... updateContact +49XXX --name "FooBar"`